TicketSystem/TicketSystem/Models/Ticket.cs

```diff
@@ -3,8 +3,8 @@
     public class Ticket
```

Comment:
For further clear understanding I recommend you name your models\entities like "TicketEntity"\"TicketModel".